account_tax_balance/models/account_move.py

```diff
@@ -0,0 +1,44 @@
+# © 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    move_type = fields.Selection(
+        string="Move type", selection=[
```

Comment:
please consider moving the selection to the separate method for better extensibility